published/024-2019-jan-8.md

```diff
@@ -0,0 +1,49 @@
+2018 IPFS in review
+
+# Milestones
+
++ January: Mozilla adds Support for Decentralization Protocols (https://blog.mozilla.org/addons/2018/01/26/extensions-firefox-59/)
+
++ February: Introduced the world to libp2p (https://blog.keep.network/introduction-to-libp2p-57ce6527babe)
```

Comment:
Can we keep tense consistent? Either "adds support" and "introduces the world" **or** "added support" and "introduced the world"

Reply:
@momack2 This first draft is my notes, I haven't done any of the writing for the sections yet, so there will be errors everywhere, sorry!